example/src/main/java/com/stripe/example/activity/PayWithGoogleActivity.kt

```diff
@@ -134,32 +142,58 @@ class PayWithGoogleActivity : AppCompatActivity() {
             } else {
                 billingDetails = null
             }
-            val paymentMethodCreateParams = PaymentMethodCreateParams.create(
-                PaymentMethodCreateParams.Card.create(
-                    stripeToken.id),
-                billingDetails)
-
-            // Now create PaymentMethod using
-            // Stripe#createPaymentMethod(paymentMethodCreateParams, callback)
+            val paymentMethodCreateParams =
+                PaymentMethodCreateParams.create(
+                    PaymentMethodCreateParams.Card.create(
+                        stripeToken.id),
+                    billingDetails)
+
+            stripe.createPaymentMethod(paymentMethodCreateParams,
+                object : ApiResultCallback<PaymentMethod> {
+                    override fun onSuccess(result: PaymentMethod) {
+                        progressBar.visibility = View.INVISIBLE
+                        Toast.makeText(this@PayWithGoogleActivity,
```

Comment:
should this be a weak reference?

Reply:
It should, I'll update in a follow-up since it's just an example app